src/components/Permalink/Permalink.js

```diff
@@ -181,6 +202,13 @@ class Permalink extends PureComponent {
     });
   }
 
```

Comment:
A function to call a function that is only used once? Keeping it simple would reduce code and increase readability.

Reply:
The function is made because calling setState directly in componentDidUpdate is restricted by eslint, even though called within a if.